src/FACT_internal.c

```diff
@@ -676,6 +678,33 @@ uint8_t FACT_INTERNAL_CreateSound(FACTCue *cue, uint16_t fadeInMS)
 			}
 		}
 
+		/* Calculate Max RPC Release Time */
+		cue->maxRpcReleaseTime = 0;
+		for (i = 0; i < newSound->sound->trackCount; i += 1)
+		{
+			for (j = 0; j < newSound->sound->tracks[i].rpcCodeCount; j += 1)
+			{
+				rpc = FACT_INTERNAL_GetRPC(
+					newSound->parentCue->parentBank->parentEngine,
+					newSound->sound->tracks[i].rpcCodes[j]
+				);
+				if (cue->parentBank->parentEngine->variables[rpc->variable].accessibility & 0x04)
+				{
+					if (FAudio_strcmp(
+						newSound->parentCue->parentBank->parentEngine->variableNames[rpc->variable],
+						"ReleaseTime"
+					) == 0 && rpc->parameter == RPC_PARAMETER_VOLUME)
```

Comment:
Does moving `rpc->parameter == RPC_PARAMETER_VOLUME` to the if statement above break anything? I'm mostly trying to avoid the `strcmp` for as many cases as possible, short of doing the RPC release check per-Sound at SoundBank parse time...

Reply:
I'll take a look and also look at the Track_Stop aspect.